src/plugins/intel_npu/src/backend/src/zero_device.cpp

```diff
@@ -46,6 +46,14 @@ ZeroDevice::ZeroDevice(const std::shared_ptr<ZeroInitStructsHolder>& initStructs
                        ze_result_to_description(retpci));
     }
 
+    /// Calculate and store device GOPS with formula: frequency * number of tiles * ops per tile
+    float gops = (device_properties.coreClockRate / powf(1000, 3)) * device_properties.numSlices *
+                 device_properties.physicalEUSimdWidth;
+    device_gops[ov::element::f32] = 0;
+    device_gops[ov::element::u8] = gops;
+    device_gops[ov::element::i8] = gops;
+    device_gops[ov::element::f16] = 0.5f * gops;
```

Comment:
for my education, why do we multiple by 0.5 for fp16?

Reply:
this is based on the indications from the Arch team